week1/project/ecommerce/src/components/categories.jsx

```diff
@@ -0,0 +1,32 @@
+import './categories.css';
+
+const Category = (props) => {
+    const {current, state, setState} = props
+    const key = current.slice(6)
+
+    const className = 'category--item' + ((key === state.pickCategory) && !state.itsEver ? 'category-sort' : '')
```

Comment:
you need to add space between to classes because you lose the css when you select the category.
you can try to indpect and take a look on DOM elements and navigate to the buttons it will give you this class name `category--itemcategory-sort` and that's why you lose any css rules for this button 

✏️need to modify✏️